modules/dev/src/tests/exec.ts

```diff
@@ -0,0 +1,30 @@
+import { TestNamespace, TestObject } from "../types";
+import { experimental, fs } from "@replit/extensions";
+import { assert } from "chai";
+
+const { exec } = experimental;
+
+const tests: TestObject = {
+  exec: async () => {
+    await exec.exec("touch test_text_file.txt");
+    const res = await fs.readFile("test_text_file.txt");
```

Comment:
i recommend appending the time to this file's name so the test can be run even if a previous iteration failed / was stopped half way

```suggestion
    let filename = `exec-test-${Date.now()}.txt`
    await exec.exec(`touch ${filename}`);
    const res = await fs.readFile(filename);
```

Reply:
I'll make sure all fs/exec tests write/create files/directories to a gitignored testing directory.  Will also create some test utils like creating a random file, etc.